.github/workflows/test-e2e-pr.yml

```
@@ -4,106 +4,175 @@
-      testComponents: ${{ steps.parseTitle.outputs.testComponents }}
-      testclis: ${{ steps.parsetestCli.outputs.testClis }}
+      # 输出三个变量供后续任务使用
+      changed_components: ${{ steps.find-changed-components.outputs.changed_components }} # 自动检测到的变更组件
+      manual_components: ${{ steps.parse-title.outputs.manual_components }} # 从PR标题手动指定的组件
+      testclis: ${{ steps.parse-test-cli.outputs.testClis }} # 测试命令列表
     steps:
-      - name: Parse Title
-        id: parseTitle
+      # 检出代码，需要完整历史以便比较变更
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)